.github/workflows/benchmark.yml

```diff
@@ -38,6 +42,27 @@ jobs:
           key: ${{ runner.os }}-${{ github.sha }}
           restore-keys: ${{ runner.os }}
 
+      - name: Get test data version
+        run: |
+          echo "IRIS_TEST_DATA_VERSION=$(wget --quiet -O - https://raw.githubusercontent.com/SciTools/iris-test-data/master/version.txt)" >> $GITHUB_ENV
```

Comment:
I'm uncomfortable that the versioning is managed differently depending on the test run in question:

https://github.com/SciTools/iris/blob/2202793568ca1eb23f1f8e9be42afae0f1383f34/.cirrus.yml#L40-L41

Also note that the use of release versions of iris-test-data was a deliberate decision taken in the last year for iris and iris-esmf-regrid. And before that we used commit hashes for some years, rather than just taking the latest. I can't recall the arguments for not using the very latest (if you're interested you might be able to follow the blame of `travis.yml` back when we used that).

Not sure if there's a sensible way to centralise this? At the very least we should make sure the versions are the same.

Reply:
It was not knowing how to centralise it that pushed me towards grabbing it out of the repo itself. Maybe it belongs in iris.config?

I can see that there will be times it's best not to use the latest version - if only so that someone committing to a different repo can't silently affect your test results

Reply:
>someone committing to a different repo can't silently affect your test results

Oh yeah of course! And we have in the past discussed putting routines in there too which would be a proper injection risk.

>It was not knowing how to centralise it that pushed me towards grabbing it out of the repo itself. Maybe it belongs in iris.config?

I think this is a temporary concern and the easy solution is the right one. #4193 suggests there are a lot of reasons for switching everything to GHA and then there are probably many more possibilities for shared caches/artifacts/etc.